buildSrc/src/main/kotlin/io/spine/internal/gradle/report/coverage/CodebaseFilter.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023, TeamDev. All rights reserved.
+ * Copyright 2024, TeamDev. All rights reserved.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This whole folder may not be needed, since we don't report coverage for this project, AFAIK.